src/main/java/com/revature/controllers/AccountController.java

```diff
@@ -56,4 +57,14 @@ public ResponseEntity<Object> insertAccount(@RequestBody Account account, @PathV
             return ResponseEntity.badRequest().body(e.getMessage());
         }
     }
+
+    @DeleteMapping("/delete/{accountId}")
```

Comment:
Can we change this to `@DeleteMapping("/id/{accountId}")`? 

Since we are using a DELETE call it won't conflict with the other calls and will keep more consistency without app. (both our GET and PATCH use `/p1/accounts/id/{accountId}` format